content/en/security/code_security/iast/_index.md

```diff
@@ -0,0 +1,125 @@
+
+To triage vulnerabilities, each vulnerability contains a brief description of the issue, including:
+
+- Impacted services.
+- Vulnerability type.
+- First detection.
+- The exact file and line number where the vulnerability was found.
+
+{{< img src="/security/application_security/code_security/vulnerability-details.png" alt="Code Security vulnerability details" style="width:100%;" >}}
+
+Each vulnerability detail includes a risk score (see screenshot below) and a severity rating: critical, high, medium, or low.
+
+The risk score is tailored to the specific runtime context, including factors such as where the vulnerability is deployed and whether the service is targeted by active attacks.
+
+{{< img src="/security/application_security/code_security/vulnerability_prioritization.png" alt="Code Security vulnerability prioritization" style="width:100%;" >}}
+
+## Remediation
+
+Datadog Code Security automatically provides the information teams need to identify where a vulnerability is in an application, from the affected filename down to the exact method and line number.
+
+{{< img src="/security/application_security/code_security/code_security_remediation.png" alt="Code Security vulnerability remediation" style="width:100%;" >}}
+
+When the [GitHub integration][7] is enabled, Code Security shows the first impacted version of a service, the commit that introduced the vulnerability, and a snippet of the vulnerable code. This information gives teams insight into where and when a vulnerability occurred and helps to prioritize their work.
+
+{{< img src="/security/application_security/code_security/vulnerability_code_snippet.png" alt="Code vulnerability snippet" style="width:100%;" >}}
+
+Detailed remediation steps are provided for each detected vulnerability.
+
+{{< img src="/security/application_security/code_security/remediation_recommendations.png" alt="Remediation recommendations" style="width:100%;" >}}
+
+Recommendations enable you to change the status of a vulnerability, assign it to a team member for review, and create a Jira issue for tracking.
+
+{{< img src="/security/application_security/code_security/vulnerability_jira_ticket.png" alt="creating a Jira ticket from a vulnerability" style="width:100%;" >}}
+
+**Note:** To create Jira issues for vulnerabilities, you must configure the Jira integration, and have the `manage_integrations` permission. For detailed instructions, see the [Jira integration][3] documentation, as well as the [Role Based Access Control][4] documentation.
+
+## Enabling Code Security
+
+To enable Code Security configure the [Datadog Tracing Library][9]. Detailed instructions for both methods can be found in the [**Security > Application Security > Settings**][10] section.
+
+If you need additional help, contact [Datadog support][11].
+
+## Disable Code Security
+
+For information on disabling Code Security, see [Disabling Code Security][12].
```

Comment:
This should be pulled out into a separate setup page similar to what we have done with SAST and SCA